Doc/library/code.rst

```diff
@@ -22,6 +22,12 @@ build applications which provide an interactive interpreter prompt.
    it defaults to a newly created dictionary with key ``'__name__'`` set to
    ``'__console__'`` and key ``'__doc__'`` set to ``None``.
 
+   Note that functions and classes objects created under an
+   :class:`!InteractiveInterpreter` instance will belong to the namespace
```

Comment:
"created under an II instance" seems awkward, but I have not suggestion yet.

Reply:
It was based on the patch by @mjdorma: https://bugs.python.org/file26917/issue8943.diff.